examples/tokio_async_request/main.rs

```diff
@@ -0,0 +1,107 @@
+use glib::clone;
+use gtk::glib::once_cell::sync::Lazy;
```

Comment:
I couldn't do this and it ends up giving errors because of the listbox, but if you instantiate it within the lambda there is no way for the ScrolledWindow to receive its value. Can you show me an example of what this implementation would look like?

error: 
![Screenshot from 2024-01-28 15-23-46](https://github.com/gtk-rs/gtk4-rs/assets/42524939/91f18217-9792-4268-9aff-9e8eb42bcc52)

Reply:
I achieved resolve